### PR TITLE
Fixes 404 for dynamic routes

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

- 
- 
- 

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .nowignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 coverage*
 /docs
 .nyc_output
+/packages/api/data/postgres
```
Comment (iameli) [inline .nowignore:4]: This could be just `/packages/api/data/`, we don't want to send the LevelDB database or whatever either

**Review by iameli — APPROVED** [review]
Tiny unimportant change, feel free to ship with that tweak :+1:

**Review by adamsoffer — APPROVED** [review]
LGTM. Thanks!

File: .nowignore (hunk 1 above)
Comment (darkdarkdragon) [reply .nowignore:4]: Sorry, missed that comment. Will add this in the next PR 😄

**Pull request merged**